tests/testthat/test_ExtractData_Soils.R

```diff
@@ -0,0 +1,267 @@
+context("Soil data extraction")
+skip_on_travis();
```

Comment:
from `?skip_on_travis`: 
> skip* functions are intended for use within test_that() blocks

Model your code on our other unit tests, e.g., `test_netCDF_functions.R` if you really need to prepare skips outside `test_that` blocks

Reply:
Looks fine otherwise, but this is not yet resolved.